src/cr/cube/cube.py

```diff
@@ -547,19 +547,30 @@ def _slice_idxs(self):
             return (0,)
         return range(len(self.dimensions[0].valid_elements))
 
+    @lazyproperty
+    def _show_missing(self):
```

Comment:
Why is this called `"_show_missing"`? It implies that we have some kind of "show" settings, which are different from context to context. Crunch-cube is strictly a 2nd order analysis library, and shouldn't be concerned with the presentation. A more appropriate name would be something that implies a "has" concept, rather than "show".